internal/pkg/aws/ec2/ec2.go

```diff
@@ -59,24 +83,49 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
+// ListVPC returns IDs of all VPCs.
+func (c *EC2) ListVPC() ([]string, error) {
```

Comment:
One thing that might be nice to do is return a struct with the VPC ID and the value of the "name" tag if it exists. This is what the console does.

Reply:
> Looks great! One follow up we can do is include the name of the Subnets/VPCs - not just the IDs.

i'll cut an issue for it!